virt/creating_vms_advanced/creating_vms_cli/virt-creating-vms-by-cloning-pvcs.adoc

```diff
@@ -31,6 +31,14 @@ This method creates a data volume whose lifecycle is independent of the original
 +
 This method creates a data volume whose lifecycle is dependent on the original VM. Deleting the original VM deletes the cloned data volume and its associated PVC.
 
+include::modules/virt-optimizing-clone-performance-at-scale-in-openshift-data-foundation.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../../../virt/storage/virt-configuring-storage-profile.adoc#virt-customizing-storage-profile_virt-configuring-storage-profile[Setting a default cloning strategy using a storage profile]
+* link:https://docs.redhat.com/en/documentation/red_hat_openshift_data_foundation/4.15/html/managing_and_allocating_storage_resources/volume-cloning_rhodf#volume-cloning_rhodf[Volume cloning]
```

Comment:
Just checking that you are linking to the 4.15 version of the docs on purpose, and not 4.18?